src/core/toga/widgets/canvas.py

```diff
@@ -99,6 +99,12 @@ def remove(self, drawing_object):
         self.drawing_objects.remove(drawing_object)
         self.redraw()
 
+    def clearAll(self):
```

Comment:
Why `clearAll`? Why not just `clear`?

Reply:
will change, i think  I was trying to differentiate between clearing a single draw command with clearing all the draw commands